### ISSUE TITLE
Add speaker template to post on twitter

### ISSUE THREAD

**Issue by marimeireles** [PyLadiesBerlin/resources#29]
I think we could advertise the people who are going to give talks during the week on twitter in a template where there's a picture of the person and some bullet points on what are their awesomeness. Then we have more chances to ad for the event + let more people know about its specificities.
I'll come up with a simple to edit template on google slides or smth and post it here.
Please let me know what you think @sleepypioneer, @discombobulateme. and idk who else to mark.

**Comment by discombobulateme** [comment]
Ohhhh I loved it!
About 2 years ago I made a GoogleSlide template, but it is time to change that too hahahha I am getting tired to see them... I'll try to get some proper designer help...

**Comment by marimeireles** [comment]
I could do something simple for the next meetup if the designer template is not ready :)

**Comment by discombobulateme** [comment]
We are using it for... a long time... changes are super welcome :D

**Comment by quosi** [comment]
Super cool, that you take on this task. Not sure if this helps, but I made a g-slides template for speakers to get started with prepering a presentation. So they don't start on a blank page, when creating there first talk. It also includes a kind of vita / profile page. Maybe that's usefull for what you want to do.
https://docs.google.com/presentation/d/1u_Ty1t_7vRoxwC60YKPk65cQnZs8DNxZpGu28nxUUM4/edit?usp=sharing

Do you have access to our PyLadies g-mail / g-drive already @marimeireles ?

**Comment by marimeireles** [comment]
Yes! This helps! Specially giving me hints on how to keep consistent with the visual identity :)
That's more or less what I had in mind, but I'd like to focus on one speaker at a time so we could give them more emphasis.

> Do you have access to our PyLadies g-mail / g-drive already @marimeireles ?

Thanks to @discombobulateme now I do. :)
